### DISCUSSION TITLE
Обход дискорда ошибка 1075 помогите

### DISCUSSION THREAD

**Discussion by forginff** [Flowseal/zapret-discord-youtube#767] [General]
уже много что пробовал чтобы исправить эту ошибку 1075 но ничего не помогает прошу помогите кто что знает
![photo_2024-11-19_22-05-59](https://github.com/user-attachments/assets/3fb46b70-18db-422d-8ca4-c31bd153de53)
[SC] StartService FAILED 1075:
The dependency service does not exist or has been marked for deletion.

**Comment by killedbyasphyxia** [comment]
С такой проблемой уже сталкивались, попробуй этот [метод](https://github.com/Flowseal/zapret-discord-youtube/issues/741#issuecomment-2481080881), типу помогло

**Reply by forginff** [reply]
мне нужно открыть именно сервисе goodbyedpi чтобы с ним работало

**Comment by Flowseal** [comment]
https://github.com/Flowseal/zapret-discord-youtube/issues/779
